src/slic3r/GUI/GCodeViewer.cpp

```diff
@@ -873,11 +775,22 @@ void GCodeViewer::init(ConfigOptionMode mode, PresetBundle* preset_bundle)
 
     // set to color print by default if use multi extruders
     if (m_nozzle_nums > 1) {
-        m_view_type_sel = std::distance(view_type_items.begin(),std::find(view_type_items.begin(), view_type_items.end(), EViewType::Summary));
-        set_view_type(EViewType::Summary);
+        m_view_type_sel = std::distance(view_type_items.begin(),std::find(view_type_items.begin(), view_type_items.end(), libvgcode::EViewType::Summary));
+        set_view_type(libvgcode::EViewType::Summary);
     } else {
-        m_view_type_sel = std::distance(view_type_items.begin(),std::find(view_type_items.begin(), view_type_items.end(), EViewType::ColorPrint));
-        set_view_type(EViewType::ColorPrint);
+        m_view_type_sel = std::distance(view_type_items.begin(),std::find(view_type_items.begin(), view_type_items.end(), libvgcode::EViewType::ColorPrint));
+        set_view_type(libvgcode::EViewType::ColorPrint);
```

Comment:
Line type by default for single extruder
```suggestion
        m_view_type_sel = std::distance(view_type_items.begin(),std::find(view_type_items.begin(), view_type_items.end(), libvgcode::EViewType::FeatureType));
        set_view_type(libvgcode::EViewType::FeatureType);
```